package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "type": "module",
+  "devDependencies": {
+    "@eslint/js": "^9.13.0",
+    "@stylistic/eslint-plugin": "^2.9.0",
+    "@stylistic/eslint-plugin-js": "^2.9.0",
+    "eslint": "^9.13.0",
+    "eslint-formatter-checkstyle": "^8.40.0",
+    "eslint-formatter-multiple": "^2.0.0",
+    "globals": "^15.11.0"
+  },
+  "scripts": {
```

Comment:
(nit) I've never seen scripts below dependencies, I normally expect to see them near the top of the file, below e.g. name